bitbots_body_behavior/src/bitbots_body_behavior/body_behavior.py

```diff
@@ -56,7 +56,7 @@
     D.blackboard.dynup_action_client = actionlib.SimpleActionClient('dynup', DynUpAction)
 
     # TODO: callbacks away from the blackboard!
-    rospy.Subscriber("balls_relative", PoseWithCertaintyArray, D.blackboard.world_model.balls_callback)
+    # rospy.Subscriber("balls_relative", PoseWithCertaintyArray, D.blackboard.world_model.balls_callback)
```

Comment:
Please remove this line if it is unused.

Reply:
done.